data.tf

```diff
@@ -45,41 +48,41 @@ data "aws_ami" "selected" {
 
   filter {
     name   = "name"
-    values = ["${var.ami_name_filter}"]
+    values = var.ami_name_filter
   }
 
   filter {
     name   = "virtualization-type"
-    values = ["${var.ami_virtualization_type_filter}"]
+    values = var.ami_virtualization_type_filter
   }
 
-  owners = ["${var.ami_image_owner}"]
+  owners = var.ami_image_owner
 }
 
 data "aws_efs_file_system" "selected" {
-  count          = "${length(var.efs_file_system) > 0 ? 1 : 0}"
-  file_system_id = "${local.efs_file_system_id}"
+  count          = length(var.efs_file_system) > 0 ? 1 : 0
+  file_system_id = local.efs_file_system_id
 }
 
-provider "template" {}
+provider "template" {
+}
```

Comment:
`terraform fmt` will still accept `provider "template" {}`. I suggest we revert it. Not sure why the migration tool changes the formatting...